roles/sap_hana_rhv_hypervisor/meta/main.yml

```
@@ -0,0 +1,56 @@
+---
+galaxy_info:
+
+  role_name: sap_hana_rhv_hypervisor
+  author: Nils Koenig
+  description: Provide the configuration for a RHV hypervisor for SAP HANA
+  company: Red Hat
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: GPL-2.0-or-later
+
+  min_ansible_version: 2.9
+
+  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # min_ansible_container_version:
+
+  #
+  # Provide a list of supported platforms, and for each platform a list of versions.
+  # If you don't wish to enumerate all versions for a particular platform, use 'all'.
+  # To view available platforms and versions (or releases), visit:
+  # https://galaxy.ansible.com/api/v1/platforms/
+  #
+  # platforms:
```

```suggestion
  platforms:
  - name: EL
    versions:
    - "8"
```
As per my understanding this role is intended for RHEL 8